adapters/teads/teads_test.go

```diff
@@ -0,0 +1,27 @@
+package teads
+
+import (
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderTeads, config.Adapter{
+		Endpoint: "https://r.teads.tv/rich/"}, config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"})
```

Comment:
should use endpoint specified in bidder-info/teads.yaml - `https://a.teads.tv/prebid-server/bid-request`